CONTRIBUTING.md

```diff
@@ -39,10 +39,55 @@ When your code is approved, it can be merged into the `dev` branch by a member o
 ## Conventions
 
 Please develop according to the following principles:
+
 - One View per file. A file containing a View struct must end in "View". We're yet to decide on an official naming scheme for files - feel free to offer your thoughts [here](https://github.com/mlemgroup/mlem/issues/55).
```

Comment:
This should be updated at some point since we appear to have settled on `TitleCase+Extension` syntax